packages/admin/admin-generator/package.json

```diff
@@ -0,0 +1,71 @@
+{
+    "name": "@comet/admin-generator",
+    "version": "7.14.0",
+    "repository": {
+        "directory": "packages/admin/admin-generator",
+        "type": "git",
+        "url": "https://github.com/vivid-planet/comet"
+    },
+    "license": "BSD-2-Clause",
+    "main": "lib/index.js",
+    "typings": "lib/index.d.ts",
+    "bin": {
+        "comet-admin-generator": "bin/admin-generator.js"
+    },
+    "files": [
+        "bin/**/*.js",
+        "lib/*"
+    ],
+    "scripts": {
+        "build": "$npm_execpath run clean && tsc -p tsconfig.build.json",
+        "clean": "rimraf lib",
+        "dev": "tsc --watch --preserveWatchOutput -p tsconfig.build.json",
+        "lint": "run-p lint:prettier lint:eslint lint:tsc",
+        "lint:eslint": "eslint --max-warnings 0 src/ **/*.json --no-warn-ignored",
+        "lint:prettier": "npx prettier --check './**/*.{js,json,md,yml,yaml}'",
+        "lint:tsc": "tsc",
+        "test": "jest --passWithNoTests",
+        "test:watch": "jest --watch"
+    },
+    "dependencies": {
+        "@graphql-tools/graphql-file-loader": "^7.5.17",
+        "@graphql-tools/load": "^7.8.14",
+        "change-case": "^4.1.2",
+        "commander": "^9.5.0",
+        "glob": "^10.4.5",
+        "object-path": "^0.11.8",
+        "pluralize": "^8.0.0"
+    },
+    "devDependencies": {
+        "@comet/admin": "workspace:*",
+        "@comet/admin-icons": "workspace:*",
+        "@comet/cms-admin": "workspace:*",
+        "@comet/eslint-config": "workspace:*",
+        "@mui/material": "^6.4.5",
+        "@mui/x-data-grid": "^7.27.0",
+        "@types/jest": "^29.5.14",
+        "@types/node": "^22.13.4",
+        "@types/object-path": "^0.11.4",
+        "@types/pluralize": "^0.0.33",
+        "@types/react": "^18.3.18",
+        "eslint": "^9.20.1",
+        "graphql": "^16.10.0",
+        "jest": "^29.7.0",
+        "npm-run-all2": "^5.0.2",
+        "prettier": "^3.5.1",
+        "react": "^18.3.1",
+        "react-intl": "^6.8.9",
+        "rimraf": "^6.0.1",
+        "ts-jest": "^29.2.5",
+        "ts-node": "^10.9.2",
+        "typescript": "^4.9.5"
+    },
+    "peerDependencies": {
+        "graphql": "^15.0.0 || ^16.0.0",
+        "ts-node": "^10.9.1"
```

Comment:
why is that a peerDependency?

Reply:
That's indeed strange; @johnnyomair and I talked about this as well. 

I think that I copied it from the cli package into api-generator package and now in the admin-generator package as well.

I think it should be possible to remove ts-node as a peer dependency from all three mentioned packages.

Reply:
If it's not a peer dependency, then it should be a normal dependency, not a dev dependency.

Reply:
i think we could also remove "graphql" as peer dependency

Reply:
@johnnyomair  agree 👍